README.md

```diff
@@ -204,3 +208,10 @@ If you are using IPFix this is likely caused by the router using an IPFix versio
 that uses timestamps relative to the system initilization time and not Unix timestamps.
 This is a known issue and we are working on a solution to support more IPFix versions.
 If this error persists, Active Countermeasures recommends switching to Netflow v9 or v5.
+
+### Log Rotation Error
```

Comment:
I would just break that sentence into 2:

It has been discovered that some flow logs will report flow
start and end times in UTC+00:00. This means if you don't live in the UTC+00:00
timezone you may see logs in MongoDB from a future date or not rotating when
you expect them to. Until a fix is implemented, Active Countermeasures asks you
to be aware of the issue.